mubench.pipeline/data/java_interface.py

```diff
@@ -0,0 +1,77 @@
+from enum import Enum, IntEnum
+from logging import Logger
+from typing import Optional, List, Dict
+
+from data.project_version import ProjectVersion
+from utils.shell import Shell
+
+def _quote(value: str):
+    return "\"{}\"".format(value)
+
+class JavaInterface:
+    def __init__(self, jar_path: str, java_options: List[str]):
+        self.jar_path = jar_path
+        self.java_options = java_options
+
+    @staticmethod
+    def get(cli_version: str, jar_path: str, java_options: List[str]) -> 'JavaInterface':
+        interfaces = JavaInterface.__subclasses__()
+        matching_interfaces = [i for i in interfaces if i.version() == cli_version]
```

Comment:
We should assert that there is only one matching interface. Otherwise, we broke something!